src/tools/wasm-component-ld/README.md

```diff
@@ -0,0 +1,62 @@
+# `wasm-component-ld`
+
+This wrapper is a wrapper around the [`wasm-component-ld`] crates.io crate. That
+crate. That crate is itself a thin wrapper around two pieces:
```

Comment:
There's an incomplete sentence here (and still seems to be there on master).

Reply:
Oh oops, I'll work on fixing this.